src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -417,8 +417,8 @@ CylindricalBinaryCompactObject::CylindricalBinaryCompactObject(
         std::array{rotate_from_z_to_x_axis(center_A_),
                    rotate_from_z_to_x_axis(center_B_)},
```

Comment:
Can't reach it with GitHub comments, but I believe there's a bug above when checking if both inner spheres and the outer sphere are specified. I think it should be

```cpp
  if (time_dependent_options_.has_value() and
      not(include_inner_sphere_A and include_inner_sphere_B
                                 and include_outer_sphere)) {
```